english/explore-outside/call-scotty/call-scotty.md

```diff
@@ -0,0 +1 @@
+You feel inside your pockets and suprisingly find a Starfleet communicator. You flip it open, press the send button and say "Beam me up, Scotty!"
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.